gradio/chat_interface.py

```diff
@@ -461,18 +461,32 @@ def _generate_chat_title(self, conversation: list[MessageDict]) -> str:
             title = title[:40] + "..."
         return title or "Conversation"
 
+    @staticmethod
+    def serialize_conversation(conversation: Any) -> list[MessageDict]:
```

Comment:
nit: might be clearer name:
```suggestion
    def serialize_cocomponents(conversation: Any) -> list[MessageDict]:
```